frontend/doc-site/032__Contribute and Publish Data.mdx

```diff
@@ -34,26 +34,28 @@ We need the following collection metadata (i.e. details associated with your pub
   - Publication/preprint DOI: can be added later
   - URLs: any additional URLs for related data or resources, such as GEO or protocols.io - can be added later
 
-Datasets need the following information added to a single h5ad (AnnData 0.7) format file per CELLxGENE visualization:
+Datasets need the following information added to a single h5ad (AnnData 0.8) format file per CELLxGENE visualization:
 
 - **Dataset-level metadata in uns**:
-  - schema_version: `2.0.0`
+  - schema_version: `3.0.0`
   - title: title of the individual dataset visualization
   - optional: batch_condition: list of obs fields that define that “batches” that a normalization or integration algorithm should be aware of
```

Comment:
In the original:

that define that "batches" that -> that define the "batches" or that define "batches" ?